kubetest/extract.go

```diff
@@ -393,7 +393,10 @@ func (e extractStrategy) Extract(project, zone string) error {
 		}
 		return getKube(url, release)
 	case gcs:
-		return getKube(path.Dir(e.option), path.Base(e.option))
+		// strip gs://foo -> /foo
+		withoutGS := e.option[3:]
```

Comment:
nit: `strings.TrimPrefix` is a more self-documenting way of achieving same.

Reply:
makes sense, I figured that since the GCS case requires this prefix format it wasn't necessary.